functions/microwin/Microwin-RemovePackages.ps1

```diff
@@ -58,7 +59,8 @@ function Microwin-RemovePackages {
                     $_ -NotLike "*VBSCRIPT*" -AND
                     $_ -NotLike "*License*" -AND
                     $_ -NotLike "*Hello-Face*" -AND
-                    $_ -NotLike "*ISE*"
+                    $_ -NotLike "*ISE*" -AND
+                    $_ -NotLike "*OpenSSH*"
```

Comment:
Will not leave the OpenSSH-Server installed as well or is only installed manualy? I don't remember.

Reply:
This will exclude everything that contains "OpenSSH". In my Windows images, there's only the client package, but it should exclude any other package that has to do with OpenSSH